### PR TITLE
MTDSA-7776 Updated ReadMe file

### PR DESCRIPTION
**Pull request by HarryBaxter**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,14 +1,42 @@
...
  
-# Run tests
+## Run Tests
 ```
 sbt test
 sbt it:test
 ```
 
-# License
+## Documentation
```
Comment (Thepworth95) [inline README.md:28]: rename to `## To view the RAML`

File: README.md [hunk 2]
```diff
@@ -1,14 +1,42 @@
...
+To view documentation locally ensure the Obligations API is running, and run api-documentation-frontend:
+`./run_local_with_dependencies.sh`
+
+Then go to http://localhost:9680/api-documentation/docs/api/preview and use this port and version:
+`http://localhost:7793/api/conf/1.0/application.raml`
+
+## API Reference / Documentation 
+Available on the [HMRC Developer Hub](https://developer.service.hmrc.gov.uk/api-documentation)
```
Comment (Thepworth95) [inline README.md:36]: change link from the Developer hub home to the Developer hub documentation for obligations

File: README.md [hunk 3]
```diff
@@ -1,14 +1,42 @@
...
+
+
+## Requirements
+- Scala 2.12.x
+- Java 8
+- sbt 1.3.13
+- [Service Manager](https://github.com/hmrc/service-manager)
+
+## Run Microservice
+To run the microservice from console, use `sbt run`
```
Comment (Thepworth95) [inline README.md:18]: add the default port in here `(starts on port 7793 by default)` at the end of the line

File: README.md [hunk 4]
```diff
@@ -1,14 +1,42 @@
...
+`./run_local_with_dependencies.sh`
+
+Then go to http://localhost:9680/api-documentation/docs/api/preview and use this port and version:
+`http://localhost:7793/api/conf/1.0/application.raml`
+
+## API Reference / Documentation 
+Available on the [HMRC Developer Hub](https://developer.service.hmrc.gov.uk/api-documentation)
+
+## Reporting Issues
+You can create a GitHub issue [here](https://github.com/hmrc/individuals-expenses-api/issues)
```
Comment (Thepworth95) [inline README.md:39]: make this a link to obligations rather than expenses

**Review by Thepworth95 — APPROVED** [review]

**Review by Nboaram — APPROVED** [review]

**Pull request merged**